### PR TITLE
MyBike

### PR DESCRIPTION
**Pull request by Timi2424**
[DEMO LINK](https://Timi2424.github.io/layout_miami/)

### CODE REVIEW

**Review by mbruhler — CHANGES_REQUESTED** [review]
Hi,
Good job on mobile version. There's still lots to do for other viewports

Also on mobile, last image (the bottom one) is too big.

**Review by mvjl000 — CHANGES_REQUESTED** [review]
- Textarea should have the same font as other inputs (currently it's monospace), send button shouldn't have default border, values of the form should clear on send
- You are missing the smooth scrolling,
- Both of these should be a call-able link 
Screenshot 2023-10-27 at 15 00 59
- Speaking of links (e.g. in the navigation) - they should have cursor pointer. All links should have cursor pointer.  In my opinion also clickable elements like buttons should have it too. It tells the user that he can interact with it. Maybe you would consider it? Tell me what you think ;)

**Comment by Timi2424** [comment]
> * Textarea should have the same font as other inputs (currently it's monospace), send button shouldn't have default border, values of the form should clear on send
> * You are missing the smooth scrolling,
> * Both of these should be a call-able link
>   Screenshot 2023-10-27 at 15 00 59
> * Speaking of links (e.g. in the navigation) - they should have cursor pointer. All links should have cursor pointer.  In my opinion also clickable elements like buttons should have it too. It tells the user that he can interact with it. Maybe you would consider it? Tell me what you think ;)

**Comment by Timi2424** [comment]
from my side, the textarea has proper font family and send button has box shadow on hover

**Review by mvjl000 — CHANGES_REQUESTED** [review]
- Smooth scrolling, contact link (from menu) should head the user to the contact section

**Review by mvjl000 — CHANGES_REQUESTED** [review]
On tablet resolution the header title should be way lower
Screenshot 2023-10-31 at 08 43 46

You are still missing cursor pointer for the links (see one of my previous reviews)

File: src/styles/blocks/body.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      opacity: 1;
+      transform: translateX(0);
+      pointer-events: auto;
+    }
+  }
+}
+
+a {
+  &:hover {
+    cursor: url(../images/cursor/cursorhover.png), auto;
```
Comment (mvjl000) [inline src/styles/blocks/body.scss:36]: Simply do:
```suggestion
    cursor: pointer;
```

**Review by mvjl000 — CHANGES_REQUESTED** [review]
About those cursors...

File: src/styles/blocks/body.scss (hunk 1 above)
Comment (mvjl000) [reply src/styles/blocks/body.scss:36]: By the way, it shouldn't be inside the hover {}

**Review by mvjl000 — CHANGES_REQUESTED** [review]
`cursor: pointer;`

**Review by mvjl000 — APPROVED** [review]

**Pull request closed without merging**